packages/gcp/src/domain/GcpFootprintEstimationConstants.ts

```diff
@@ -18,7 +18,7 @@ import {
 export const GCP_CLOUD_CONSTANTS: CloudConstantsByProvider = {
   SSDCOEFFICIENT: 1.2, // watt hours / terabyte hour
   HDDCOEFFICIENT: 0.65, // watt hours / terabyte hour
-  MIN_WATTS_MEDIAN: 0.71,
+  MIN_WATTS_MEDIAN: 0.68,
```

Comment:
If we're adding support for additional features, should we be editing these numeric constants? Will this affect how we deal with other GCP data?